src/routes.js

```diff
@@ -2,7 +2,7 @@ const routes = require("next-routes")();
 
 routes
   .add("shopProduct", "/shop/:shopSlug/product/:productSlug", "product")
-  .add("product", "/product/:productSlug", "product")
+  .add("product", "/product/:productSlug/:variantId", "product")
```

Comment:
`:variantId` needs to be optional or there needs to be another route for it as I cannot enter the PDP page without messing with the URL.